### PR TITLE
Replaced the adaptive chunking code for Flash with a constant one with a bandwidth of roughly 1MB/s

### PR DESCRIPTION
**Pull request by imbcmdth**
The purpose of this code was to upload to Flash as quickly as possible. The intent was noble but it never quite managed to work without introducing jankiness at higher bitrates.

### CODE REVIEW

**Comment by imbcmdth** [comment]
Required for https://github.com/videojs/videojs-contrib-hls/pull/918

File: src/flash-constants.js [hunk 1]
```diff
@@ -4,36 +4,17 @@
...
- * as much as possible in TIME_PER_TICK and while tuning the size of appends
- * dynamically so that we only append about 4-times in that 4ms span.
- *
- * The reason we try to keep the number of appends around four is due to
- * externalities such as Flash load and garbage collection that are highly
- * variable and having 4 iterations allows us to exit the loop early if
- * an iteration takes longer than expected.
+ * we chunk data and pass through 4KB at a time, yielding to the
+ * browser between chunks. This gives a theoretical maximum rate of
+ * 1MB/s into Flash. Any higher and we being to drop frames and UI
```
Comment (gesinger) [inline src/flash-constants.js]: s/being/begin

File: src/flash-source-buffer.js [hunk 2]
```diff
@@ -260,58 +252,35 @@ export default class FlashSourceBuffer extends videojs.EventTarget {
...
     }
-
-    // We also make sure that the chunk-size doesn't drop below 1KB or
-    // go above 1MB as a sanity check
-    this.chunkSize_ = Math.max(
-      FlashConstants.MIN_CHUNK,
-      Math.min(this.chunkSize_, FlashConstants.MAX_CHUNK));
+    let b64str = window.btoa(binary);
+
+    // bypass normal ExternalInterface calls and pass xml directly
```
Comment (gesinger) [inline src/flash-source-buffer.js]: That's crazy to me that this is faster

**Comment by mjneil** [comment]
LGTM

File: src/flash-source-buffer.js (hunk 2 above)
Comment (imbcmdth) [inline src/flash-source-buffer.js]: Yeah... it was back in the day. I don't know if the performance of ExternalInterface calls have improved in recent times.

**Comment by gesinger** [comment]
LGTM

**Pull request merged**